src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Ball {
+    String color;
```

Comment:
The field 'color' should have an access modifier to encapsulate the internal representation of the object. Consider using 'private' to adhere to the principle of encapsulation.